build-system/babel-plugins/babel-plugin-transform-amp-asserts/index.js

```diff
@@ -94,20 +138,21 @@ module.exports = function(babel) {
       }
     }
 
-    path.replaceWith(t.parenthesizedExpression(arg));
+    if (path.parentPath.isExpressionStatement() && isPure(argument)) {
```

Comment:
Can you verify whether this is still needed?

Reply:
Which part? Are you suggesting that the entire block can be removed?

Reply:
Yes.

Reply:
I've removed this block, removed the now unused `isPure()` function you added earlier, and re-generated the output files for `gulp babel-plugin-tests`. Let me know what you think.

Reply:
What's the size delta now?

Reply:
From https://github.com/ampproject/amphtml/pull/24047/checks?check_run_id=391871742, it appears to be -0.16KB (size reduction) for `v0.js`

Reply:
This is moot. Will post the latest diffs in a fresh comment thread after rebasing.